Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -5772,3 +5772,279 @@ Download the provided software version onto the device.
 >| 11111111111111 | true |
 >| 192.168.1.145 | true |
 
+
+### pan-os-platform-reboot
+***
+Reboot the given device by hostid. Warning; this command has no confirmation and the device
+will immediately reboot.
+
+
+#### Base Command
+
+`pan-os-platform-reboot`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| hostid | ID of host (serial or hostname) to reboot. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| PANOS.RestartStatus.Summary.hostid | String | Host ID | 
+| PANOS.RestartStatus.Summary.started | String | Whether download process has started. | 
```

Comment:
can you add context example? make sure you add it for all the rest of the commands (you have already added it to some of the commands). 

Also add human readable section if possible.

Reply:
Yeah, the ones I didn't add it to I couldn't build command examples for. I added context output for these but no human readable because it's a pain to write the markdown manually. Can add if absolutely required.

Reply:
if its not too much of effort, it could be nice to have a simple example.